### DISCUSSION TITLE
Add PLA-CF

### DISCUSSION THREAD

**Discussion by nickglott** [MarksMakerSpace/filament-profiles#17] [Ideas]
Can you add PLA-CF to Materials?

**Comment by markcarroll** [comment]
There is already a CF sub-type so set the type to PLA and sub.-type to CF.  Will that not work? I am wondering whether to rethink the split.

**Comment by nickglott** [comment]
Oh OK! I mean that works, I did not even look cause I saw PET-CF so I just assumed.

**Reply by markcarroll** [reply]
Ah.  Probably need to clean that up a bit then.
